### PR TITLE
Include events for Application Members

### PR DESCRIPTION
**Pull request by dimitrisnl**
Connects-To: #36
Change-Type: minor

### CODE REVIEW

File: packages/resin-event-log/src/resin-event-log.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ var EVENTS = {
 		'delete'
 	],
 	applicationTag: [ 'set', 'create', 'edit', 'delete' ],
+	applicationMembers: [ 'set', 'edit', 'delete' ],
```
Comment (thgreasi) [inline packages/resin-event-log/src/resin-event-log.js]: I would say that it's create edit delete

**Comment by dimitrisnl** [comment]
Updated 👍

**Review by thgreasi — APPROVED** [review]

**Comment by thgreasi** [comment]
Should we also add events about device parking while we are here?

**Comment by dimitrisnl** [comment]
I would make sense to have a separate issue.

**Comment by dimitrisnl** [comment]
@thgreasi Actually there is one
And we *do* track it in the ui!

**Review by pimterry — APPROVED** [review]

**Pull request merged**